docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -71,3 +71,16 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+[WARNING]
+
+In IPv6, {OpenShift} worker nodes need a `/64` prefix allocation due to OVN
+limitations (RFC 4291). For dynamic IPv6 configuration, you need to change the
+prefix allocation on the Router Advertisement settings. If you want to use
+manual configuration for IPv6, define a similar CR to the
+`NodeNetworkConfigurationPolicy` CR example in this procedure, and define an
+IPv6 address and disable IPv4.  Because the constraint for the `/64` prefix did
+not exist in {OpenstackPreviousInstaller}, your previous {OpenStackShort}
+control plane network might not have enough capacity to allocate these
+networks. If that is the case, allocate a prefix that fits large enough number
+of addresses. Depends on the number of worker nodes you have, /60 might be enough.
```

Comment:
```suggestion
IPv6 address and disable IPv4. Because the constraint for the `/64` prefix did
not exist in {OpenstackPreviousInstaller}, your previous {OpenStackShort}
control plane network might not have enough capacity to allocate these
networks. If that is the case, allocate a prefix that fits a large enough number
of addresses, for example, `/60`. The prefix depends on the number of worker nodes you have.
```

Reply:
Just some minor edits. Otherwise, looks good!

Reply:
Thanks, done! I also added the conditional suggested by Jiri.